engine/pong.h

```diff
@@ -43,6 +47,8 @@ class Pong
         }
         return nullptr;
     }
+
+    static void SetTimeout(int gameObjectId, int timeoutMs, std::function<void()> callback);
```

Comment:
Nice use of std::function.

Reply:
Yeah. It is really nice that this seemingly can callback any function with any amount of args using std::bind or a lambda? Makes something like this real easy to implement.